docs/api/web-frame-main.md

```diff
@@ -122,6 +122,14 @@ content. The identifier is fixed at the creation of the frame and stays
 constant for the lifetime of the frame. When the frame is removed, the id is
 not used again.
 
+#### `frame.name` _Readonly_
+
+A `String` representing the frame name.
+
+#### `frame.pid` _Readonly_
+
+An `Integer` representing the operating system `pid` of the process which owns this frame.
```

Comment:
Can we update the docs for `frame.processId` to clarify that it's referencing Chromium's internal process ID? The web contents docs mention this so it's likely helpful here as well https://www.electronjs.org/docs/api/web-contents#contentsgetosprocessid

Reply:
It may also be worth changed `frame.pid` to `frame.osProcessId` for consistency with `WebContent`.

Reply:
done